proms-resources/clinical-data/src/main/resources/SLING-INF/content/Extensions/Sidebar/PMCC.json

```
@@ -0,0 +1,8 @@
+{
+  "jcr:primaryType": "cards:Extension",
+  "cards:extensionPointId": "cards/coreUI/sidebar/entry",
+  "cards:extensionName": "PMCC",
+  "cards:targetURL": "/UserDashboard/Cardio",
+  "cards:icon": "asset:cards-dataentry.cardioIcon.js",
```

The icon asset is defined in the `proms-homepage` module
```suggestion
  "cards:icon": "asset:proms-homepage.cardioIcon.js",
```